pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: stripe_payment
 description: A Flutter plugin to integrate the stripe plugin for iOS and Android. Currently only adding a credit card as payment method is implemented.
 version: 0.3.1
```

Comment:
can you adjust the version number?

Reply:
You got it, changed in 'bump yaml version'